src/Model/ActivityList/DefaultMariaDbActivityList/MariaDbDao.php

```diff
@@ -100,7 +100,7 @@ private function addLimit(QueryBuilder $select)
     public function getCount()
     {
         $query = $this->getQueryBuilder();
-        $query->setFirstResult(null);
+        $query->setFirstResult(0);
```

Comment:
@blankse Thank you for the fixes! Looks all good.
I've only noticed that `MySqlDao.php` change would be missing in this PR, if we are changing it here for MariaDb
https://github.com/pimcore/customer-data-framework/blob/35a4f04c638321ba86732296e91b7419cf461fff/src/Model/ActivityList/DefaultMariaDbActivityList/MySqlDao.php#L106
(although i see that both are covered in #311 in the minor release branch)

Reply:
Done in https://github.com/pimcore/customer-data-framework/pull/327/commits/dbe349cb54e0728ac8465787d51f9cc32c9da4a1